_jugs/SevillaJUG.md

```diff
@@ -0,0 +1,8 @@
+---
+name:     "Sevilla Java User Group"
+country:  Spain
+website:  https://www.meetup.com/SVQJUG/
+twitter:  /
```

Comment:
It will be much better if you simply remove the "Twitter entry" if you don't have a Twitter account 😉 
But, it could be that it is this one? https://twitter.com/SVQJUG

Reply:
```suggestion
twitter:  SVQJUG
```